tests/integration/test_uninstall.py

```diff
@@ -44,90 +65,89 @@ def test_mirror_uninstall(PipenvInstance, pypi):
         )
         assert "pypi.org" not in mirror_url
 
-        c = p.pipenv("install requests --pypi-mirror {0}".format(mirror_url))
+        c = p.pipenv("install Django==1.11.13 --pypi-mirror {0}".format(mirror_url))
         assert c.return_code == 0
-        assert "requests" in p.pipfile["packages"]
-        assert "requests" in p.lockfile["default"]
-        assert "chardet" in p.lockfile["default"]
-        assert "idna" in p.lockfile["default"]
-        assert "urllib3" in p.lockfile["default"]
-        assert "certifi" in p.lockfile["default"]
+        assert "django" in p.pipfile["packages"]
+        assert "django" in p.lockfile["default"]
+        assert "pytz" in p.lockfile["default"]
         # Ensure the --pypi-mirror parameter hasn't altered the Pipfile or Pipfile.lock sources
         assert len(p.pipfile["source"]) == 1
         assert len(p.lockfile["_meta"]["sources"]) == 1
         assert "https://pypi.org/simple" == p.pipfile["source"][0]["url"]
         assert "https://pypi.org/simple" == p.lockfile["_meta"]["sources"][0]["url"]
 
-        c = p.pipenv("uninstall requests --pypi-mirror {0}".format(mirror_url))
+        c = p.pipenv("run python -m django --version")
         assert c.return_code == 0
-        assert "requests" not in p.pipfile["dev-packages"]
-        assert "requests" not in p.lockfile["develop"]
-        assert "chardet" not in p.lockfile["develop"]
-        assert "idna" not in p.lockfile["develop"]
-        assert "urllib3" not in p.lockfile["develop"]
-        assert "certifi" not in p.lockfile["develop"]
+
+        c = p.pipenv("uninstall Django --pypi-mirror {0}".format(mirror_url))
+        assert c.return_code == 0
+        assert "django" not in p.pipfile["dev-packages"]
+        assert "django" not in p.lockfile["develop"]
+        assert p.lockfile["develop"] == {}
         # Ensure the --pypi-mirror parameter hasn't altered the Pipfile or Pipfile.lock sources
         assert len(p.pipfile["source"]) == 1
         assert len(p.lockfile["_meta"]["sources"]) == 1
         assert "https://pypi.org/simple" == p.pipfile["source"][0]["url"]
         assert "https://pypi.org/simple" == p.lockfile["_meta"]["sources"][0]["url"]
 
-        c = p.pipenv("run python -m requests.help")
+        c = p.pipenv("run python -m django --version")
```

Comment:
Note Django was used here to continue using the `run python -m  ..` syntax, and there are not many packages in `tests/pypi/` that have a `__main__`.  We could add another lighter package with a `__main__` to the pypi archive .

Or this could be changed to using `run python -c 'import x'` which is almost as good, with a slightly less understandable error if the test fails.  But I think 99% of the people likely to encounter an error here are going to understand the backtrace caused by `import x` failing.